cms/templates/manage_users_lib.html

```diff
@@ -0,0 +1,181 @@
+          <div class="wrapper-form">
+            <h3 class="title">${_("Grant Access to This Library")}</h3>
+
+            <fieldset class="form-fields">
+              <legend class="sr">${_("New Team Member Information")}</legend>
+
+                <ol class="list-input">
+                  <li class="field text required create-user-email">
+                    <label for="user-email-input">${_("User's Email Address")}</label>
+                    <input id="user-email-input" class="user-email-input" name="user-email" type="text" placeholder="${_('example: username@domain.com')}" value="">
+                    <span class="tip tip-stacked">${_("Please provide the email address of the user you'd like to add")}</span>
+                  </li>
+                </ol>
+            </fieldset>
+          </div>
+
+          <div class="actions">
+            <button class="action action-primary" type="submit">${_("Add User")}</button>
+            <button class="action action-secondary action-cancel">${_("Cancel")}</button>
+          </div>
+        </form>
+      </div>
+      %endif
+
+      <ol class="user-list">
+        % for user in all_users:
+          <%
+          is_instructor = user in instructors
+          is_staff = user in staff
+          role_id = 'admin' if is_instructor else ('staff' if is_staff else 'user')
+          role_desc = _("Admin") if is_instructor else (_("Staff") if is_staff else _("User"))
+          %>
+
+        <li class="user-item" data-email="${user.email}">
+
+          <span class="wrapper-ui-badge">
+            <span class="flag flag-role flag-role-${role_id} is-hanging">
+              <span class="label sr">${_("Current Role:")}</span>
+              <span class="value">
+                ${role_desc}
+                % if request.user.id == user.id:
+                    <span class="msg-you">${_("You!")}</span>
+                % endif
+              </span>
+            </span>
+          </span>
+
+          <div class="item-metadata">
+            <h3 class="user-name">
+              <span class="user-username">${user.username}</span>
+              <span class="user-email">
+                <a class="action action-email" href="mailto:${user.email}" title="${_("send an email message to {email}").format(email=user.email)}">${user.email}</a>
+              </span>
+            </h3>
+          </div>
+
+          % if allow_actions:
+          <ul class="item-actions user-actions">
+            % if is_instructor:
+              % if len(instructors) > 1:
+                <li class="action action-role">
+                  <a href="#" class="make-staff admin-role remove-admin-role">${_("Remove Admin Access")}</span></a>
+                </li>
+              % else:
+                <li class="action action-role">
+                  <span class="admin-role notoggleforyou">${_("Promote another member to Admin to remove admin rights")}</span>
```

Comment:
Suggest making this text identical to the equivalent on the Course Team settings page in Studio. Just add the word "your" after "remove": "Promote another member to Admin to remove your admin rights."